cockroachcloud/frequently-asked-questions.md

```diff
@@ -23,26 +25,29 @@ This page answers the frequently asked questions about {{ site.data.products.ser
- `australia-southeast1` (Australia)  | `ap-southeast-1` (Singapore)    
- `europe-west1` (St. Ghislain)       | `ap-southeast-2` (Sydney)       
- `europe-west2` (London)             | `ca-central-1` (Central Canada) 
- `europe-west3` (Frankfurt)          | `eu-central-1` (Frankfurt)      
- `europe-west4` (Eemshaven)          | `eu-north-1` (Stockholm)        
- `northamerica-northeast1` (Montréal)| `eu-west-1` (Ireland)           
- `southamerica-east1` (São Paulo)    | `eu-west-2` (London)            
- `us-central1` (Iowa)                | `eu-west-3` (Paris)             
- `us-east1` (South Carolina)         | `sa-east-1` (São Paulo)         
- `us-west1` (Oregon)                 | `us-east-2` (Ohio)              
- `us-west2` (California)             | `us-west-2` (Oregon)            
-
-We run {{ site.data.products.dedicated }} in EKS and GKE - the managed Kubernetes offerings for AWS and GCP respectively - and support all regions in which their offerings are available. If a particular region is not available on the {{ site.data.products.db }} Console, it is usually due to the cloud provider not supporting EKS or GKE in that region. See
-[list of EKS regions](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services/) and [list of GKE regions](https://cloud.google.com/about/locations/) for details.
+ GCP                                 | AWS                               | Azure (Limited Access)
+-------------------------------------|-----------------------------------|-----------------------
+ `asia-east1` (Changhua County)      | `ap-northeast-1` (Tokyo)          | `eastus2` (US East Coast - Virginia)
+ `asia-east2` (Hong Kong)            | `ap-northeast-2` (Seoul)          | `westeurope` (Netherlands)
+ `asia-northeast1` (Tokyo)           | `ap-northeast-3` (Osaka)
+ `asia-southeast1` (Jurong West)     | `ap-south-1` (Mumbai)
+ `australia-southeast1` (Australia)  | `ap-southeast-1` (Singapore)
+ `europe-west1` (St. Ghislain)       | `ap-southeast-2` (Sydney)
+ `europe-west2` (London)             | `ca-central-1` (Central Canada)
+ `europe-west3` (Frankfurt)          | `eu-central-1` (Frankfurt)
+ `europe-west4` (Eemshaven)          | `eu-north-1` (Stockholm)
+ `northamerica-northeast1` (Montréal)| `eu-west-1` (Ireland)
+ `southamerica-east1` (São Paulo)    | `eu-west-2` (London)
+ `us-central1` (Iowa)                | `eu-west-3` (Paris)
+ `us-east1` (South Carolina)         | `sa-east-1` (São Paulo)
+ `us-west1` (Oregon)                 | `us-east-2` (Ohio)
+ `us-west2` (California)             | `us-west-2` (Oregon)
+
+We run {{ site.data.products.dedicated }} in EKS, GKE, or AKS, the managed Kubernetes offerings from AWS, GCP, and Azure, and generally support all regions in which their offerings are available. If a particular region is not available on the {{ site.data.products.db }} Console, it may be because the cloud provider not supporting EKS, GKE, or AKS in that region. Refer to the following:
```

Comment:
It's not true. We don't support regions not just coz k8s service is not available there, but also coz we don't want to make it available anywhere without the demand. I think this para needs bit of a change to indicate that we support regions that are required by our customers and we keep adding more based on demand.

Reply:
Yeah I had misgivings about this. PTAL at what I have done for now, remembering that there is a bigger work item to consolidate where we discuss regions to a central page.